client/src/components/EventCountdown.vue

```diff
@@ -0,0 +1,63 @@
+<!-- hardcoded countdown for ProtoTrip, should be made adjustable in the future -->
+<template>
+  <div
+    class="border-proto_blue dark:bg-proto_secondary_gray-dark absolute top-0 left-0 mt-3 ml-4 w-2/5 rounded-lg border-l-4 bg-white pr-4 font-medium text-gray-900 shadow-lg ring-1 ring-black ring-opacity-5 dark:text-gray-50"
+    style="
+      background: url(https://static.saproto.com/image/32733/d8368c638dbb4c5e5cab9f0b8631d9d4fba34be2?w=800&h=300);
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-position: center center;
+    ">
+    <div
+      class="dark:bg-proto_secondary_gray-dark/50 m-2 flex h-full w-full flex-col content-center rounded-lg bg-white/50 px-4">
+      <h1 class="text-center text-3xl" style="text-shadow: #fc0 1px 1px 10px">
+        ProtoTrip 2023 location reveal & sign up
+      </h1>
+      <div v-if="!countdownOver" class="m-2 flex justify-center" id="countdown">
+        <ul class="flex justify-center px-3">
+          <li class="mx-3 text-4xl" style="text-shadow: #fc0 1px 1px 10px">
+            <span></span>{{ hours }} hours
+          </li>
+          <li class="mx-3 text-4xl" style="text-shadow: #fc0 1px 1px 10px">
+            <span></span>{{ minutes }} minutes
+          </li>
+          <li class="mx-3 text-4xl" style="text-shadow: #fc0 1px 1px 10px">
+            <span></span>{{ seconds }} seconds
+          </li>
+        </ul>
+      </div>
+      <!-- if any of you spoil the location i will royeer you -->
+      <h1
+        v-else
+        class="m-2 animate-bounce text-center text-4xl"
+        style="text-shadow: #fc0 1px 1px 10px">
+        We will venture to Aarhus, Denmark!
```

Comment:
Je zou t evt in base64 kunnen zetten mocht iemand de pagina inspecteren dat t niet direct leesbaar is?